test/unit/test_default_inference_handler.py

```diff
@@ -24,11 +24,20 @@ def test_default_input_fn(loads):
     loads.assert_called_with(42, content_types.JSON)
 
 
+@pytest.mark.parametrize(
+    "accept, expected_content_type",
+    [
+        ("text/csv", "text/csv"),
+        ("text/csv, application/json", "text/csv"),
+        ("unsupported/type, text/csv", "text/csv"),
+        ("unsupported/type", "application/json"),
+    ],
+)
```

Comment:
Not sure if I should split this up into separate functions. Benefit would be that I could use descriptive test names describing the intent of the test, but adding more functions might be unnecessarily verbose.

Reply:
what split were you considering?

Reply:
Separate test for each pair of `accept` and `expected_content_type` arguments.

From https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices, the scenario under test and the expected behavior should be obvious from the name. But with a parameterized test, we're testing multiple scenarios. Not sure how applicable these practices are for Python, though

Reply:
I think what you have here is fine